### PR TITLE
Add lightning-bolts via grayskull

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/lightning-bolts/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lightning-bolts-{{ version }}.tar.gz
+  sha256: ea65874469e204950837c4998870c87ff1e725668bbb424f8b6d8e91dcda2fdf
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (beenje) [inline recipes/lightning-bolts/meta.yaml:15]: This is a pure Python package. It should be `noarch: python`.
See [Noarch python](https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=noarch#noarch-python)

File: recipes/lightning-bolts/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (beenje) [inline recipes/lightning-bolts/meta.yaml:20]: For noarch package, put a minimum version on Python: `>=3.6`

File: recipes/lightning-bolts/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    - wheel
+  run:
+    - dataclasses  # [py<=36]
```
Comment (beenje) [inline recipes/lightning-bolts/meta.yaml:23]: You can't use selector for noarch packages. You don't have to for `dataclasses`.
The conda package will be empty for python >=3.7. See https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=noarch#empty-python-packages

File: recipes/lightning-bolts/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - wheel
```
Comment (beenje) [inline recipes/lightning-bolts/meta.yaml:21]: I don't think you have to specify `wheel`.

**Comment by Tobias-Fischer** [comment]
Thanks for the review @beenje! Would be great if you could take another look, I think I addressed all comments.

**Comment by beenje** [comment]
@conda-forge/help-python, this PR looks good to me.

It's failing on windows due to missing dependencies but it's a noarch.

**Pull request merged**